client/build/asset-manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "files": {
+    "main.css": "/static/css/main.ba49ec77.chunk.css",
+    "main.js": "/static/js/main.40d0fa47.chunk.js",
+    "main.js.map": "/static/js/main.40d0fa47.chunk.js.map",
+    "runtime~main.js": "/static/js/runtime~main.a8a9905a.js",
+    "runtime~main.js.map": "/static/js/runtime~main.a8a9905a.js.map",
+    "static/js/2.0c3a1eaf.chunk.js": "/static/js/2.0c3a1eaf.chunk.js",
+    "static/js/2.0c3a1eaf.chunk.js.map": "/static/js/2.0c3a1eaf.chunk.js.map",
+    "index.html": "/index.html",
+    "precache-manifest.8f2ca5a06d893ad07c0d94c6ae0e4791.js": "/precache-manifest.8f2ca5a06d893ad07c0d94c6ae0e4791.js",
+    "service-worker.js": "/service-worker.js",
+    "static/css/main.ba49ec77.chunk.css.map": "/static/css/main.ba49ec77.chunk.css.map"
+  }
+}
```

Comment:
Is it possible to not push these changes since we gitignored it? You make need to look at:
https://intellipaat.com/community/2837/git-stop-tracking-file-how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore

Reply:
It really clouds up this PR, making it harder to review

Reply:
Yes, sorry I've reverted that commit now.